HPXMLtoOpenStudio/measure.rb

```diff
@@ -632,7 +632,16 @@ def self.add_floors(runner, model, spaces)
       surface.setName(floor.id)
       if floor.is_interior
         surface.setSunExposure('NoSun')
-        surface.setWindExposure('NoWind')
+        if floor.exterior_adjacent_to == HPXML::LocationManufacturedHomeUnderBelly
+          foundation = @hpxml.foundations.find {|foundation| foundation.to_location == floor.exterior_adjacent_to}
+          if foundation.belly_wing_skirt_present
+            surface.setWindExposure('NoWind')
+          else
+            surface.setWindExposure('WindExposed')
+          end
```

Comment:
@shorowit regarding our conversation earlier about not requiring a `Foundation` element for OpenStudio-HPXML: I'm using it here to determine whether the under belly should have exposure to wind. I guess that'd mean we need a default assumption about the skirt if there's no `Foundation` available. What do you think that should be?

Reply:
I'd probably assume there's a skirt.